main.js

```diff
@@ -257,3 +253,57 @@ function winnerNameOutput() {
     winnerName.innerText = twoName.value;
   }
 }
+
+// function below will reset game to default with new generated correct number
+document.addEventListener('keyup', enableReset);
+resetButton.addEventListener('click', resetGame);
+
+function enableReset() {
+  var hasMin = minRange.value !== "";
+  var hasMax = maxRange.value !== "";
+  var hasOneName = oneName.value !== "";
+  var hasTwoName = twoName.value !== "";
+  var hasOneGuess = oneGuess.value !== "";
+  var hasTwoGuess = twoGuess.value !== "";
+  var isFilled = hasMin || hasMax || hasOneName || hasTwoName || hasOneGuess || hasTwoGuess;
+  if (isFilled) {
+    resetButton.disabled = false;
+  } else {
+    resetButton.disabled = true;
+  }
+}
```

Comment:
This will definitely work, but I'm wondering if there isn't a better way to enable it?  Right now it only enables if you have a value in an input, but what if it enables if they've changed the range, submitted any guesses, etc?  Just food for thought.